neuromation/cli/admin.py

```diff
@@ -26,4 +26,31 @@ def admin() -> None:
     )
 
 
+@command()
+@click.argument("cluster_name", required=True, type=str)
+@click.argument("user_name", required=True, type=str)
+@click.argument("role", required=False, default="user", type=str)
+@async_cmd()
+async def add_cluster_user(
+    root: Root, cluster_name: str, user_name: str, role: str
+) -> None:
+    """
+    Add user access to specified cluster with one of 3 roles: admin, manager or user
+    """
+    await root.client._admin.add_cluster_user(cluster_name, user_name, role)
+
+
+@command()
+@click.argument("cluster_name", required=True, type=str)
+@click.argument("user_name", required=True, type=str)
+@async_cmd()
+async def remove_cluster_user(root: Root, cluster_name: str, user_name: str) -> None:
+    """
+    Remove user access from the cluster
+    """
+    await root.client._admin.remove_cluster_user(cluster_name, user_name)
```

Comment:
Should we print something to the user, like: 
Added user: Ivan[admin]
Removed user: Alex

Reply:
I think yes.
Ideally, we should print texts as you had in examples in the normal mode and skip printing if `--quiet` is specified.
Adding ANSI escapes (bold, underline, etc.) can improve visual readability, see `neuro config show` and others for inspiration.

Reply:
Added echo calls with quiet support